docs/tutorial/index.html

```diff
@@ -17,6 +17,194 @@
 <header id="title-block-header">
 <h1 class="title">Testo tutorial</h1>
 </header>
-<p>Coming soon!</p>
+<h2 id="introduction">Introduction</h2>
+<p>Testo is a test framework for OCaml. Like with OUnit or Alcotest, the user writes a collection of tests. A test consists of a name and an OCaml test function to run, with some options. If the test function returns, the test is considered successful but if it raises an exception, it is considered failed.</p>
+<p>The test suite is compiled into a test executable with a fancy command-line interface provided by the Testo library.</p>
```

Comment:
I would replace "fancy CLI" by something describing what this CLI offer in few words
(like ability to run only certain test, snapshot update).